Makefile

```diff
@@ -9,4 +9,7 @@ build:
 	bundle exec jekyll build --future --drafts --unpublished
 
 test:
+	## Check for broken Markdown reference-style links that are displayed in text unchanged, e.g. [broken][broken link]
+	! find _site/ -name '*.html' | xargs grep ']\[' | grep -v skip-test | grep .
```

Comment:
Is the `grep -v skip-test` necessary?

Reply:
Not yet here, but it's been used to solve issues in other repositories:

```text
user@devbco:~/bitcoincore.org$ git grep skip-test
Makefile:       ! find _site/ -name '*.html' | xargs grep ']\[' | grep -v skip-test | grep .
_releases/0.14.0.md:`"proxy"`      | `getnetworkinfo()["networks"][0]["proxy"]` <!--skip-test-->
_releases/0.14.0.md:array index or a non-quoted string (i.e. `listunspent()[0][txid]`). <!--skip-test--> Both
```

Reply:
Makes sense!